src/themes/themes.ts

```diff
@@ -4,11 +4,11 @@
  * Map of theme Javascript modules
  */
 export const themes: {
-  elife: Promise<any>
-  nature: Promise<any>
-  plos: Promise<any>
-  skeleton: Promise<any>
-  stencila: Promise<any>
+  elife: Promise<unknown>
```

Comment:
Are these changes correct?

Reply:
If it compiles then yes it's fine and `unknown` is indeed better than `any`.